cmd/devnet/main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
We could certainly instantiate a `devnet` by using the lotus APIs and libraries directly in Go, but I think it is best if we test the dealbot with an actual version of the `lotus` binaries. WDYT?

Reply:
Doing it all in pure Go would have the advantage that we don't need the user to do a separate build of lotus, and it would also give us the pinning of the lotus version for free. So I think it makes more sense in the long run. It can certainly be left as a TODO here, though.

Reply:
I am not sure I want the dealbot to be responsible for building lotus, lotus-miner and lotus-seed - especially when it is not a simple build process - it involves filecoin-ffi, etc.

I do agree that having the integration test itself in go test would be nice - that could be done on next iteration.